meetup/meetup.rb

```diff
@@ -0,0 +1,34 @@
+=begin
+Write your code for the 'Meetup' exercise in this file. Make the tests in
+`meetup_test.rb` pass.
+
+To get started with TDD, see the `README.md` file in your
+`ruby/meetup` directory.
+=end
+
+require 'date'
+
+WEEKDAYS = [:sunday, :monday, :tuesday, :wednesday, :thursday, :friday, :saturday]
+ORDINALS = [:first, :second, :third, :fourth, :fifth] 
+
+class Meetup
+
+    def initialize(month, year)
+      @month = month
+      @year = year
+    end
+  
+    def day(weekday, which)
+      matches = (1..Date.new(@year, @month, -1).day).select{ |d| Date.new(@year, @month, d).wday == WEEKDAYS.index(weekday) }
+      if which == :last
+        day = matches[-1]
+      elsif which == :teenth
+        day = matches.select{ |d| d > 12 }[0]
+      else
+        day = matches[ORDINALS.index(which)]
+      end
+      return Date.new(@year, @month, day) 
```

Comment:
return may be omitted here, because method returns the last line expression by default

Reply:
Sorry. I know, but I used to using it))